qiskit/__init__.py

```diff
@@ -56,18 +56,21 @@
 try:
     from qiskit.providers.aer import Aer
 except ImportError:
-    if _config.get('suppress_packaging_warnings') or os.environ.get(
-            'QISKIT_SUPPRESS_PACKAGING_WARNINGS', '').upper() != 'Y':
+    suppress_warnings = os.environ.get('QISKIT_SUPPRESS_PACKAGING_WARNINGS', '')
+    if not _config.get('suppress_packaging_warnings') or (_config.get(
+            'suppress_packaging_warnings') and
+            suppress_warnings.upper() != 'N') or suppress_warnings.upper() != 'Y':
```

Comment:
There's a lint failure here. Also, I think this will always show warnings even if set in the config.

Reply:
Heh, I think pylint has gotten confused by the nested if structure:

`Wrong continued indentation (add 46 spaces).`

I guess I'll have to restructure this as nested ifs instead of one

Reply:
This should be resolved by: https://github.com/Qiskit/qiskit-terra/pull/3362/commits/a4e29d96cd926f06e4997c7780d31a6d70792b58